tests/testthat/test-transform_point_model_out.R

```diff
@@ -123,17 +123,16 @@ test_that("many-to-one relationship exists between model_out_tbl and oracle_outp
 })
 
 test_that("hubExamples data set is transformed correctly", {
-  # forecast_outputs.rda & forecast_oracle_output.rda are stored in hubExamples:
-  # https://github.com/hubverse-org/hubExamples/tree/main
-  model_out_tbl <- hubex_forecast_outputs()
-  oracle_output <- hubex_forecast_oracle_output()
+  # Forecast data from hubExamples: <https://hubverse-org.github.io/hubExamples/reference/forecast_data.html>
+  forecast_outputs <- hubExamples::forecast_outputs
```

Comment:
changed names from `model_out_tbl` --> `forecast_outputs` and `oracle_output` --> `forecast_oracle_output` for consistency with other tests using these data objects from hubExamples.